packages/shared/src/components/auth/ProviderButton.tsx

```diff
@@ -24,7 +24,6 @@ function ProviderButton({
       style={{
         justifyContent: label ? 'flex-start' : 'center',
         color: '#FFFFFF',
-        border: 0,
```

Comment:
I think the reason we had this as an inline style was to override the button's base class for border. Though if the `className` can do that no issues, then we can move forward with this.